daemon/src/maker_cfd_actor.rs

```diff
@@ -37,11 +39,13 @@ pub fn new<B, D>(
     takers: mpsc::UnboundedSender<maker_inc_connections_actor::Command>,
     cfd_feed_actor_inbox: watch::Sender<Vec<Cfd>>,
     order_feed_sender: watch::Sender<Option<Order>>,
+    wallet_feed_sender: watch::Sender<WalletInfo>,
 ) -> (
     impl Future<Output = ()>,
     mpsc::UnboundedSender<maker_cfd_actor::Command>,
 )
 where
+    B: Blockchain,
     D: BatchDatabase,
```

Comment:
Needs a rebase on top of @klochowicz work, otherwise master will fail once you merge this.

brb activating bors ...